plugins/woocommerce/changelog/52993-dev-52933-payments-settings-api-refactor

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: tweak
+Comment: This is just refactoring existing code for the Payments NOX project.
```

Comment:
Small nit: maybe something like `Refactor of existing backend code for the Reactified Payments Settings, no functional changes` reads better as a changelog entry, since it'll be shown in a long list of other changes.

Reply:
Ok. Changing it, but that is only a comment on why there isn't a changelog entry.